src/element-stub.js

```diff
@@ -33,3 +43,50 @@ export class ElementStub extends BaseElement {
     return true;
   }
 }
+
+// function to calculate scriptSrc from location path
+export function calculateScript(path) {
+  let scriptSrc;
+  if (getMode().localDev) {
+    scriptSrc = 'https://cdn.ampproject.org/v0/amp-ad-0.1.js';
+    if (path.search('max') >= 0) {
```

Comment:
We typically use `indexOf` instead of `search` with strings (instead of reg exes).